components/countdown/Countdown.tsx

```diff
@@ -0,0 +1,104 @@
+'use client';
+
+import EventDate, { getDateFromEvent } from '@/models/EventDate';
+import {
+    CSSProperties,
+    ReactElement,
+    useCallback,
+    useEffect,
+    useState,
+} from 'react';
+
+interface Time {
+    years: number;
+    days: number;
+    hours: number;
+    minutes: number;
+    seconds: number;
+}
+
+export default function Countdown({
+    event,
+}: {
+    event: EventDate;
+}): ReactElement {
+    const calcTime = useCallback((): Time => {
```

Comment:
Maybe consider this lib https://day.js.org/

Reply:
> Maybe consider this lib https://day.js.org/

Do you have any tips? Or can I just leave it as it is for now?